src/isolate.rs

```diff
@@ -1560,9 +1579,35 @@ impl Drop for OwnedIsolate {
         "v8::OwnedIsolate instances must be dropped in the reverse order of creation. They are entered upon creation and exited upon being dropped."
       );
       self.exit();
-      self.cxx_isolate.as_mut().clear_scope_and_annex();
-      self.cxx_isolate.as_mut().dispose();
+      self.dispose_scope_root();
+      self.dispose_annex();
+      self.dispose();
+    }
+  }
+}
+
+impl OwnedIsolate {
```

Comment:
this moved up so destroying an isolate to create a blob was near where isolates are normally destroyed, I can move it back down if wanted though.